idd/Energy+.idd.in

```diff
@@ -51383,6 +51383,10 @@ Coil:Cooling:DX:CurveFit:Performance,
        \note is used as Reheat mode.
        \type object-list
        \object-list DXCoolingOperatingModeNames
+   A8; \field Outdoor Temperature Dependent Crankcase Heater Capacity Curve Name
```

Comment:
To be consistent with other curve names this should be
"Crankcase Heater Capacity Function of Temperature Curve Name"
ref. 
Coil:Cooling:DX:SingleSpeed, Total Cooling Capacity Function of Temperature Curve Name
Coil:Cooling:DX:CurveFit:Speed, Total Cooling Capacity Modifier Function of Temperature Curve Name

Also this should be tagged with 
```
       \type object-list
       \object-list UnivariateFunctions
```

Reply:
Right, I'll change the name to "Crankcase Heater Capacity Function of Temperature Curve Name" to match the existing patterns and add the object-list tag.

Reply:
right right, I'll change min-fields too for ones too for those with the new curve field not beyond the current min-field.